src/main/scala/fastq.scala

```diff
@@ -92,43 +188,47 @@ case object fastq {
       rec(raw, Seq.empty, false)
     }
 
-    val toPhred33: Int => Char = i => (i + 33).toChar
+    val toPhred33: Int => Char = { i => (i + 33).toChar }
   }
 
-  // TODO value should not have @ at the beginning
-  class Id(val value: String) extends AnyVal {
+  class Id private[fastarious] (val value: String) extends AnyVal {
 
     def asString: String =
       s"@${value}"
   }
+
   case object Id {
 
-    private val isValid: String => Boolean =
-      raw =>
-        (raw startsWith "@") &&
-        !(raw contains "\n")
+    private val isValid: String => Boolean = { raw =>
+      (raw startsWith "@") &&
+      !(raw contains "\n")
+    }
 
     def from(raw: String): Option[Id] = {
 
       // just *one* line
+      // FIXME: raw starts with several @, it will drop them all
       val l = raw.filterNot(_ == '\n').dropWhile(_ == '@')
```

Comment:
- it will just join several strings in one
- and drop any number of @ in the beginning